streams-activitypub-webapp/src/test/scala/org/apache/streams/activitypub/webapp/test/ActivityPubWebappTestSuiteExtension.scala

```diff
@@ -81,24 +76,13 @@ class ActivityPubWebappTestSuiteExtension extends ParameterResolver with BeforeA
 
   override def afterAll(extensionContext: ExtensionContext): Unit = {
     LOGGER.info("afterAll()")
-    try {
-      tomcat.stop();
-    } catch {
-      case e: Exception => LOGGER.error("Exception while stopping tomcat", e);
-    } finally {
-      try {
-        tomcat.destroy();
-      } catch {
-        case e: Exception => LOGGER.error("Exception while destoying tomcat", e);
-      }
-    }
   }
```

Comment:
if BeforeAllCallback and AfterAllCallback could be activated only by the 'TestSuite' starts and stops vs the test classes that constitute the class, this code should return.  couldn't figure out how to make this happen yet though